.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*' # semver release tags
+      - 'v*.*.*-*' # pre-release tags for testing
+
+permissions:
+  contents: write # needed by goreleaser/goreleaser-action for publishing release artifacts
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+        with:
+          go-version: '1.19'
+
+      - name: Import GPG Key
+        id: import_gpg
+        uses: crazy-max/ghaction-import-gpg@72b6676b71ab476b77e676928516f6982eef7a41 # v5.3.0
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
```

We already have a secret for the passphrase, which is called:

```suggestion
          passphrase: ${{ secrets. PASSPHRASE }}
```

I could regenerate, but I don't know the existing data that was used so I'd rather just re-use it.

---

Ah okay that’s easy enough to change. Thanks for calling that out. 

Is it okay for me to squash all the changes into a single commit after this change? Or would you prefer I keep them separate?

---

Squashing is great

---

Great! All should be good to go now. Thanks again for working through these changes with me.